controllers/v1/usersController.js

```diff
@@ -588,6 +590,179 @@ module.exports = class Users {
+                        "status": "started",
+                        "title": "Observation 1"
+                    }
+                ],
+                [
+                    {
+                        "_id": "60d58ee5983c1372591ccb5b",
+                        "entityId": "b54a5c6d-98be-4313-af1c-33040b1703aa",
+                        "entityInformation": {
+                            "name": "VIZIANAGARAM"
+                        },
+                        "entityType": "district",
+                        "createdBy": "fca2925f-1eee-4654-9177-fece3fd6afc9",
+                        "status": "completed",
+                        "title": "Observation 1"
+                    }
+                ],
+                [
+                    {
+                        "_id": "60caf262ecea0772d81d7e5f",
+                        "entityId": "2f76dcf5-e43b-4f71-a3f2-c8f19e1fce03",
+                        "entityInformation": {
+                            "name": "ANANTAPUR"
+                        },
+                        "entityType": "district",
+                        "createdBy": "fca2925f-1eee-4654-9177-fece3fd6afc9",
+                        "status": "started",
+                        "title": "Observation 1"
+                    }
+                ]
+            ]
+        }]}
+     * @apiUse successBody
+     * @apiUse errorBody
+     */
+     
+    /**
+    * list Observation Info
+    * @method
+    * @name listObservationInfo
+    * @param {Object} req -request Data.
+    * @returns {JSON} - List of observation consumed by an user
+    */
+
+    listObservationInfo(req) {
+        return new Promise(async (resolve, reject) => {
+            try {
+    
+                let queryObject = {
+                    createdBy: req.userDetails.userId,
+                }
+    
+                let stats = req.query.stats;
+                
+                let fields = [
+                    'name',
+                    'isAPrivateProgram',
+                    'entities',
+                    'status',
+                    'description',
+    
+                ] 
+    
+                if (stats == "true") {
+                  fields = ["_id"];
+                }
+                let observationDocument =
+                  await observationsHelper.observationDocuments(
+                    queryObject,
+                    fields
+                  );
+                if (stats !== "true") {
+                  for (let i = 0; i < observationDocument.length; i++) {
+                    let submissionInfoArray = [];
+                    let observationId = observationDocument[i]._id;
+                    let entityArray = observationDocument[i].entities;
+    
+                    for (let j = 0; j < entityArray.length; j++) {
+                      let entityId = entityArray[j];
+                      let observationSubmissions =
+                        await observationSubmissionsHelper.observationSubmissionsDocument(
```

Comment:
check if you can use aggregator function here or not

Reply:
avoiding this for now as I have not worked on this and this would extend my deadline